doc/development_process.md

```diff
@@ -0,0 +1,185 @@
+* **Senior developers** have merge permissions with Cleverraven and have been around a long time.  This is a very fuzzy role, none of us know exactly what makes a person a "senior" developer versus any other kind of "developer".  It just kind of happens, usually because they're doing leadership things and not being told to stop.  I-am-Erk seems to be in this role for some reason, though he can't really figure out how it happened nor why he hasn't run away yet.  Some of the senior devs have gold names on Discord, but because we love confusion, most do not.
+* **Developers** are members of Cleverraven with merge permissions.  These folks are trusted enough to be allowed the very dangerous ability to merge to the main project line.  They are the main workforce of the game's management team.  Developers have green names on discord.
+* **Collaborators** are contributors that have been around a while, and shown that they "get" the direction of the project.  We mark their names blue on discord to indicate this; the purpose of the role is to help newer users to identify if the feedback they're getting is from a fellow new person, or from someone with a bit more experience.  Collaborators don't speak for Cleverraven, necessarily, but most of the time they know what they're talking about.
+* **The dev team** is a super vague term that means, roughly, "the developers and sometimes the collaborators too".
+* **Contributors** are people who have added something to the game, anything.  Code, art, translations, writing, are all included.  While this is the most numerous group, we also think it's the most important one.  Everyone from Kevin onward is a contributor.  Contributors have purple names on discord.
+* **Core contributors** is another super vague term that we use sometimes, meaning "the developers, the collaborators, and a bunch of the contributors that have been around a lot".
+* **Players** are the people whose lives we try to ruin, who somehow keep coming back for more.  What is their deal?  We may never know.
+
+## Helpful tips for prospective contributors
+
+This section is going to be super long, and you're welcome to read it all if you like, but here's the most important TL;DR points:
+
+1.  Most simple, reasonable contributions get accepted.  For larger projects, you might want to get more experience with the project or clear it with a developer first.
+2.  If it sounds like your small job is being turned into a big unmanageable job, check if the ideas being thrown around are things *you* have to do, or just people getting excited.  Usually it's just excitement, it means you have a cool idea but not that you need to do a bunch more stuff.
+3.  If you're getting negative feedback or being told you have to do something a certain way, make sure it's someone with authority before you start feeling discouraged.
```

Comment:
This is missing a "and make sure the certain way is actually backed up by any of our guidelines". Be it code style or something else.
Reword at will.